hw3/card.js

```diff
@@ -0,0 +1,32 @@
+function Card(name, newCardBalance){          
+    this.name = name;
+    isNaN(newCardBalance)?newCardBalance=0:true;
+    this.name?true:this.name="Anonim Card";       
+    // chekcBalance function    
+    this.checkBalance = function (){        
```

Comment:
if the function returns only value - better to use name with `get` prefix. E.g. `getBalance`.
The `check` prefix mostly used when we use some condition inside and should returns `boolean` type or `Error`. E.g. 
``` js
function checkIsNumber(a){
  if(typeof a === 'number' && !Number.isNaN(a)){
   return true;
  }
  return false;
}
```

Reply:
Fixed